database/README.md

```diff
@@ -0,0 +1,168 @@
+# Database instructions
+
+## Requirements
+
+* Kubernetes cluster
+  
+  Follow the instructions to deploy a Kubernetes cluster from [Local Deployment]([kubernetes](https://kubernetes.io/docs/tasks/tools/#kubectl)).
+* go
+  
+  [Download and install go](https://go.dev/doc/install)
+* psql
+  ```bash
+  sudo dnf install postgresql
+  psql --version
+  psql (PostgreSQL) 15.6
+  ```
+* DBeaver Community
+  
+  A SQL client is recommended to interact easily with the Database. 
```

Comment:
```suggestion
  A SQL client is recommended to interact easily with the database: 
```

Database should not be capitalized here, and I think you probably want to end this with a colon, since you have the DBeaver Community title and link immediately after it.

Reply:
updated this and the others :-)